oscar/run_vqa.py

```diff
@@ -36,7 +36,7 @@
 
 log_json = []
 debug_size = 500
-
+local_rank = int(os.environ["LOCAL_RANK"])
```

Comment:
can you add a check if LOCAL_RANK exists so that it doesn't throw

Reply:
Thanks @guschmue. I have added a default value of 0 to avoid exception, https://github.com/guschmue/Oscar/pull/4/commits/925b24bf8c163de2c24ec3f312d630b23112559f